### DISCUSSION TITLE
PBPK Modelling - Neonates

### DISCUSSION THREAD

**Discussion by GabyWa** [Open-Systems-Pharmacology/Forum#1222] [Questions & Problems]
Dear community,

I have questions regarding the derived organ volumes (L) displayed in PKSim for reference subjects as also reflected in the source publication ICRP2002 and the GFR. 

Assuming the organ and tissue masses (g) provided in the ICRP and assuming the density values to derive the corresponding values in L (1.3 g/cm3 for bone, 1 g/cm3 for muscle and 0.92 < 1 for fat/adipose tissue) the resulting organ values (L) to me appear not fully in line with what can be calculated from ICRP2002 (the given source). See also the attached screenshot. 
For example, ICRP stated for a male adult reference subject the following organ masses: skeleton (10500 g), muscle (2900 g),  fat (14600 g) PKSim displays as organ volumes 11.82L for bone, 32.64L for muscle and 14.66L for fat (overall greater that expected having ICRP as source data). 
How are the organ volumes derived in PKSim - in general and in particular for the very young subjects (density values might change)? I would be very thankful to get to know the assumptions used for these calculations and your help in this regard.

This also leads to my second question regarding GFR.
When letting the physiology being displayed  for a neonate reference subject in PKSim, GFR is stated to be 3.95 ml/min (with no further reference) and a specific GFR  (10.27 ml/min/100g organ) is in addition provided with source publication (Schlender et al, 2016) that however is more focussed to the elderly population. I am wondering whether maturation will be considered that way, when one would use this for simulations for the neonate population. How could this be achieved (considering allometry + maturation) in the PKSim framework? 
Thank you for any reply, thoughts and discussion on these topics. 
![PKSim](https://user-images.githubusercontent.com/106730431/171589682-5c5a437e-b8d4-40f0-a70e-22e26d1c9bea.jpg)

**Comment by prvmalik** [comment]
If you export a simulation to Mobi, you can see the density parameter that is used for each organ.

The derivation of the GFR (mL/min) in PK-Sim that includes both Maturation and Aging is posted here https://github.com/Open-Systems-Pharmacology/Forum/discussions/791

Maturation of the GFR in neonates is assigned by the Rhodin et al. method.

**Comment by Incei** [comment]
Hi GabyWa, 

Regarding tissue density, in OSP, irrespective of age, tissue density is standard set to 1 g/cm3 in all organ tissues. It can be found using the search function in MoBi using ‘density’ in the search term. The tissue density parameter is located in the spatial structures building block, and within this building block under ‘organism’ you can find all organs under which the tissue density parameter is stored. 
 
It is assumed that tissue density is not really different than 1 g/cm3 irrespective of the organ tissue or age, that is used for e.g. the calculation of the total body weight (sum of all organs volumes * tissue density). ([Creating Populations - Open Systems Pharmacology (open-systems-pharmacology.org)](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdocs.open-systems-pharmacology.org%2Fworking-with-pk-sim%2Fpk-sim-documentation%2Fpk-sim-creating-populations&data=05%7C01%7C%7C6b312ede79094420563f08da6414e7d2%7Cfcb2b37b5da0466b9b830014b67a7c78%7C0%7C0%7C637932339669687094%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=as636dhnt7EzjNjif1v46I50pPJzK8iWbQvYgGAfdbg%3D&reserved=0))
 
The calculated weights of children that resulted from the simulations using OSP has been demonstrated to adequately describe of the pharmacokinetics of drugs from preterm newborns to adolescents. ([https://pubmed.ncbi.nlm.nih.gov/34185905/](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fpubmed.ncbi.nlm.nih.gov%2F34185905%2F&data=05%7C01%7C%7C6b312ede79094420563f08da6414e7d2%7Cfcb2b37b5da0466b9b830014b67a7c78%7C0%7C0%7C637932339669687094%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=QBMXF0gFn2cSmvwRvjjrNGEpKHU3LIfdr07jpgRk0DQ%3D&reserved=0)) The software does include age dependent information on fraction fat and muscle volumes in the interstitial space, whereas for other organs it is age-independent, and is used for the simulation of newborn children. This has however not been used to calculated tissue densities. More detailed information on tissue fractions and organ weights was published in 2006 by Edginton et al. ([https://link.springer.com/article/10.2165/00003088-200645100-00005](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Flink.springer.com%2Farticle%2F10.2165%2F00003088-200645100-00005&data=05%7C01%7C%7C6b312ede79094420563f08da6414e7d2%7Cfcb2b37b5da0466b9b830014b67a7c78%7C0%7C0%7C637932339669687094%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C2000%7C%7C%7C&sdata=DOlSXn8LCJKhetne7W6HhpIk5YshM%2F4XtefnUairjQY%3D&reserved=0)).
